initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh

```diff
@@ -35,6 +35,8 @@ export LEGACY_CI_CEXEC_LOCATION_ON_HOST="/home/shippable/cexec"
 export DEFAULT_TASK_CONTAINER_MOUNTS="-v $BUILD_DIR:$BUILD_DIR \
   -v $REQEXEC_DIR:/reqExec"
 export DEFAULT_TASK_CONTAINER_OPTIONS="-d --rm"
+export LEGACY_CI_DOCKER_CLIENT_LEGACY="/usr/bin/docker"
```

Comment:
We can remove legacy and assume that this is the latest. We're installing Docker 17.06 here.

Reply:
@shrivara wont' we need this if old users start running builds on this platform. im talking about running legacy ci running on dynamic nodes?

Reply:
Yes, but we've moved this out from boot.sh to here. So, we'll know exactly which version of Docker we're installing.

Reply:
ah. ok. so the point here is that these two variables should only be exported based on which version of docker we're running. if its legacy then only export legacy stuff else import latest stuff right?

Reply:
Yes